frame/staking/src/lib.rs

```diff
@@ -785,6 +785,18 @@ pub trait Config: frame_system::Config + SendTransactionTypes<Call<Self>> {
 	/// their reward. This used to limit the i/o cost for the nominator payout.
 	type MaxNominatorRewardedPerValidator: Get<u32>;
 
+	/// The maximum number of validators.
+	type MaxValidators: Get<u32>;
+
+	/// The maximum number of nominators.
+	type MaxNominators: Get<u32>;
+
+	/// The minimum bonded to be a validator.
+	type MinValidatorBond: Get<BalanceOf<Self>>;
```

Comment:
Just reading up until here, it is unclear what are the distinction of this and `MinBond`. Doc could be made better about when each is being used.